packages/runner/src/runtime.ts

```diff
@@ -87,11 +87,13 @@ export interface SpaceCellContents {
   allCharms: Cell<never>[];
   recentCharms: Cell<never>[];
   defaultPattern: Cell<never>;
+  favorites: Cell<Cell<never>[]>;
```

Comment:
We should maybe make a HomeSpaceCellContent type since this is only valid there? (And arguably the others here aren't relevant to people accessing the home space from other spaces)